plugin/trino-hive/src/test/java/io/trino/plugin/hive/optimizer/TestHivePlans.java

```diff
@@ -203,19 +203,19 @@ public void testSubsumePartitionPartOfAFilter()
         // Join is important because it triggers PredicatePushDown logic (EffectivePredicateExtractor)
         assertDistributedPlan(
                 "SELECT l.str_col, r.str_col FROM table_int_partitioned l JOIN table_unpartitioned r ON l.int_part = r.int_col " +
-                        "WHERE l.int_part BETWEEN 2 AND 4 AND l.str_col != 'three'",
+                        "WHERE l.int_part IN (1, 4) AND l.str_col != 'three'",
```

Comment:
Why change in an existing test?
Can we rather have an expplicit regression test for https://github.com/trinodb/trino/issues/10532 ?